ios/sdk/src/JitsiMeetManager/JitsiMeetManager.swift

```diff
@@ -0,0 +1,101 @@
+//  Copyright © 2018 Jitsi. All rights reserved.
+
+import Foundation
+
+/// Creates and coordinates the presentation of JitsiMeetViewController inside of an external window
+/// which can be resized and dragged with custom PiP mode
+open class JitsiMeetManager: NSObject {
+    
+    /// Defines if welcome screen should be on
+    public var welcomeScreenEnabled: Bool = false {
+        didSet {
+            meetViewController?.jitsiMeetView.welcomePageEnabled = welcomeScreenEnabled
+        }
+    }
+    
+    private(set) var meetViewController: JitsiMeetViewController?
+    private(set) var meetWindow: PiPWindow?
+    
+    override public init() {
+        super.init()
+        self.meetViewController = makeMeetViewController()
+        self.meetWindow = makeMeetWindow()
+    }
+    
+    public init(meetViewController: JitsiMeetViewController? = nil, meetWindow: PiPWindow? = nil) {
+        super.init()
+        self.meetViewController = meetViewController ?? makeMeetViewController()
+        self.meetWindow = meetWindow ?? makeMeetWindow()
+    }
+    
+    /// Presents and loads a jitsi meet view
+    ///
+    /// - Parameter url: The url of the presentation
+    public func load(withUrl url: URL?) {
```

Comment:
What's with the `load(withUrl` and `load(withUrlObject` names? We do call these `loadURL`, `loadURLObject', and `loadURLString` in `JitsiMeetView` and I'm not very comfortable using slight variations. We could've called them all `load` in Java/on Android because Java has proper overloading but we did want to keep a parallel between the Objective-C and Java APIs for teams which develop the two platforms. So I'd like to know why we have slight variations in Jitsi Meet SDK now for the Swift classes of this PR.